src/polyfill.js

```diff
@@ -0,0 +1,35 @@
+import Vue from 'vue'
+
+export default new Vue({
+  data () {
+    return {
+      locale: 'en',
+    }
+  },
+  watch: {
+    locale (locale) {
+      if (!window.IntlPolyfill) return
+      switch (locale) {
+        case 'de': return import('intl/locale-data/jsonp/de')
```

Comment:
I would prefer to have it so it accepts all languages. Keeps the cognitive overhead low when adding another locale. 

`import ('intl/locale-data/jsonp/${locale}')`

Anecdote: I needed to update 3 locations in the old angular codebase when adding another language and I routinely kept forgetting at least one of them. Made it quite annoying. As this happens quite rarely, it's hard to get a routine there.

Reply:
There are 719 locales available for intl, and including them all (plus all the date-fns) ones means we have close to 1000 built files in webpack. This slows things down (although I did not measure it).

Another idea I had (because I agree it's annoying and easy to forget something if you have to edit 3 things) is to have a file per locale which would load everything needed for it (karrot, date-fns, intl (if needed)).

Would having to add one file with 3 things in it lower the cognitive overhead sufficiently?

Reply:
>  I routinely kept forgetting at least one of them
> As this happens quite rarely

We can also just add a test to make sure everything can be defined.

Reply:
> Would having to add one file with 3 things in it lower the cognitive overhead sufficiently?

> We can also just add a test to make sure everything can be defined.

Both of them seem good solutions, will make an issue out of it.